src/validation.js

```diff
@@ -398,7 +398,9 @@ const validation = {
       windowsFilePath = helpers.resolveWindowsEnvironmentVariables(windowsFilePath);
       windowsFilePath = this.resolvePATH(windowsFilePath);
```

Comment:
* There needs to be some basic safety checking for this feature (see suggestion below).
* Also the `isLink` should be checked as a optional boolean defaulting to false in the validation file.
* There needs to be unit tests created to ensure the repo stays at 100% test coverage.
* The documentation in the README needs updated as well to convey this new feature.

```suggestion
      if (options.windows.isLink) {
        if (
          typeof(windowsFilePath) !== 'string' ||
          !windowsFilePath.includes('://')
        ) {
          helpers.throwError(options, 'WINDOWS filePath must be a string of a URL when isLink is true:' + windowsFilePath);
          delete options.windows;
        }
        return options;
      }
      windowsFilePath = helpers.resolveWindowsEnvironmentVariables(windowsFilePath);
      windowsFilePath = this.resolvePATH(windowsFilePath);
```